main.cc

```diff
@@ -537,6 +553,11 @@ void print_starting_message(int ac, char** av, const bpo::parsed_options& opts)
     fmt::print("parsed command line options: {}\n", format_parsed_options(opts.options));
 }
 
+static auto safe_start(stop_signal& stop, sstring what, bool ready = false) {
```

Comment:
this is not starting anything, this is a checkpoint

Reply:
what name would you suggest instead?

Reply:
checkpoint

BTW please spell out the return type, don't use `auto`

Reply:
Done in v2 ([b6705ad](https://github.com/scylladb/scylladb/commit/b6705ad48bc3aebb0a6b8fed495bbc8e30906535))